.pre-commit-config.yaml

```diff
@@ -25,3 +25,14 @@ repos:
     rev: 3.9.2
     hooks:
       - id: flake8
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 1.5.0
+    hooks:
+      - id: nbqa-black
+        additional_dependencies: [jupytext, black]
```

Comment:
you probably also want
```
  types_or: [jupyter, md]
  types: [text]  # overwrite types: [jupyter]
```

I'll fix this upstream, will push out a 1.5.1

Reply:
Is this still required?

I notice that all the .md Myst Jupytext notebooks are skipped still.

Reply:
Is this still required?

I notice that all the .md Myst Jupytext notebooks are skipped still.

Reply:
I'm surprised, I've put `types_or: [jupyter, markdown]` in all the hooks

https://github.com/nbQA-dev/nbQA/blob/main/.pre-commit-hooks.yaml

Reply:
Oh actually the pre-commit CI shows that it does work. Not sure why locally these changes weren’t made.